.gitignore

```diff
@@ -1,3 +1,4 @@
 index.html
 README.html
 out
+index.adoc.bak
```

Comment:
Welches Tool produziert `.bak` Dateien der `index.adoc`?

Reply:
Vielleicht war ich das auch selbst ^^ bin mir grad nicht sicher

Reply:
Vielleicht sollte das dann besser in eine Computer-eigene ignore-Datei und nicht in die projektspezifische. Was meinst du?